### ISSUE TITLE
Export multi-transform to ace-evaluator

### ISSUE THREAD

**Issue by cortner** [ACEsuit/ACE1.jl#22]
Not obvious to me where we should put our efforts: 
- write an export routine + an import routine and them re-implement this transform in C++
- just hook David's C interface into LAMMPs directly?

**Comment by cortner** [comment]
@gabor1 your view would be useful here.

**Comment by gabor1** [comment]
I am worried about running lammps with julia on HPCs. It *could* be that it is the future, but it would be nice not to be totally dependent on this, at least in the short run. IT would also be much less work to reimplement the transforms in PACE.

**Comment by cortner** [comment]
I don't think it is a quick hack to put the multi-transform into PACE. that's why I'm suggesting to think of both options. But we will take a look.

**Comment by cortner** [comment]
@WillBaldwin0 - does your spline export take care of this?

**Comment by WillBaldwin0** [comment]
My splines export only does the pair potential at the moment, since I haven't been messing around with the full ACE for repulsion stuff, so no multi transform

**Comment by cortner** [comment]
ok. but in principle it could be expanded to that, right?

**Comment by cortner** [comment]
but we leave it open and discuss it at one of the next software meetings.

**Comment by WillBaldwin0** [comment]
Yes it could, I'm not sure how that works on the PACE end though

**Comment by cortner** [comment]
I think the PACE end is already done that way.

**Comment by cortner** [comment]
just following up on this, I think we are going splines on everything so looking into exporting that to PACE is now urgent.
